app/src/main/java/ar/com/wolox/android/example/ui/training/homePage/ProfileFragment.kt

```diff
@@ -0,0 +1,17 @@
+package ar.com.wolox.android.example.ui.training.homePage
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+/**
+ * LoginFragment is used for user to Login
+ */
+class ProfileFragment @Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    override fun init() {
+    }
+
+    override fun layout(): Int = ar.com.wolox.android.R.layout.profile_fragment_page
```

Comment:
Please remember to follow id's nomeclature: fragment_profile_page could be better